### ISSUE TITLE
Let Tensorboard logs appear next to each other instead of in separate rows

### ISSUE THREAD

**Issue by mpaepper** [PyTorchLightning/pytorch-lightning#2070]
Is it possible to have many panels next to each other in Tensorboard?

Before using PyTorch Lightning (used Torchbearer), the panels were all next to each other.
In PyTorch Lightning, they appear underneath each other (see screenshot attached).

Is it somehow possible to group all the panels together, so they are shown next to each other?

I have many metrics and it's very hard to look at them when each one starts a new row.

### Code

Currently I do it like this:

```
    def validation_epoch_end(self, outputs):
        avg_loss = torch.stack([x['val_loss'] for x in outputs]).mean()
        tensorboard_logs = {name: torch.stack([x['metr'][name] for x in outputs]).mean() for name in self.metrics}
        tensorboard_logs['val_loss'] = avg_loss

        return {'val_loss': avg_loss, 'log': tensorboard_logs}
```

![Auswahl_172](https://user-images.githubusercontent.com/4135790/83737518-67ab5700-a653-11ea-83a6-7706382a22f9.png)

**Comment by pvnieo** [comment]
Hi there,
Tensorboard put by default panels separated, everyone in a new row. If you want some panels to be in the same row, you should give them the same prefix in the panel name, for example: "train/loss_cell" and "train/classification accuracy".

**Comment by mpaepper** [comment]
Awesome, thank you so much @pvnieo 

That works great!
